driver-core/src/test/functional/com/mongodb/ClusterFixture.java

```diff
@@ -515,6 +521,11 @@ public static boolean isAuthenticated() {
         return getConnectionString().getCredential() != null;
     }
 
+    public static boolean isClientSideEncryptionTest() {
```

Comment:
I had to exclude some additional CSFLE tests and realized we had this snippet of code duplicated all over the place, so abstracted into this helper method and updated all the tests that had this snippet.